docs/core/tools/telemetry.md

```diff
@@ -93,6 +93,20 @@ The telemetry feature collects the following data:
 | >=6.0.300     | If the CLI was invoked from a Continuous Integration environment. For more information, see [Continuous Integration Detection](#continuous-integration-detection).|
 | >=7.0.100     | Hashed PublishAot used for build (MSBuild property) |
 | >=7.0.100     | Hashed PublishProtocol used for build (MSBuild property) |
+| >=8.0.100     | Hashed TargetPlatformIdentifier used for build (MSBuild property) |
+| >=8.0.100     | Hashed RuntimeIdentifier used for build (MSBuild property) |
+| >=8.0.100     | Hashed BlazorWasm used for build (MSBuild property) |
+| >=8.0.100     | Hashed WasmSDK used for build (MSBuild property) |
+| >=8.0.100     | Hashed UsesMaui used for build (MSBuild property) |
+| >=8.0.100     | Hashed UsesMobileSDKOnly used for build (MSBuild property) |
+| >=8.0.100     | Hashed UsesOtherMobileSDK used for build (MSBuild property) |
+| >=8.0.100     | Hashed MonoAOT used for build (MSBuild property) |
+| >=8.0.100     | Hashed NativeAOT used for build (MSBuild property) |
+| >=8.0.100     | Hashed Interp used for build (MSBuild property) |
+| >=8.0.100     | Hashed LibraryMode used for build (MSBuild property)|
+| >=8.0.100     | Hashed HybridGlobalization used for build (MSBuild property) |
+| >=8.0.100     | Hashed ResolvedRuntimePack used for build (MSBuild property) |
+| >=8.0.100     | Hashed StripILAfterAOT used for build (MSBuild property) |
```

Comment:
@baronfel @steveisok Please correct me if I am wrong but, I don't think all these should be included. 

My understanding is that we should include only the properties available/configurable by the end user, and not what we internally set/determine during the build. 

For example from: https://github.com/dotnet/runtime/pull/90208 and
```xml
<WorkloadPublishProperties Include="NativeAOT" Value="$(_WorkloadUsesNativeAOT)" />
...
<_WorkloadUsesNativeAOT>$(PublishAot)</_WorkloadUsesNativeAOT>
```
we should just enlist that `PublishAot` value is considered in data collection and add into this table.
While `NativeAOT` is a property we can fetch when we analyse the usage, am I correct?

Reply:
While `NativeAot` is redundant, its value comes into play when querying. There's a specific event we publish all the workload data under.

Reply:
> While NativeAot is redundant, its value comes into play when querying. There's a specific event we publish all the workload data under.

I see, but afaik this table should include the data points we collect and not what will be used for querying, right?

Reply:
Guess it depends. It is a duplicate field, but is tracked nonetheless. I don't have a strong opinion one way or another.

Reply:
On a similar note, should we document that `UsesOtherMobileSDK` actually checks for usage of Avalonia and Uno ?

Reply:
The minimum declared here should be the 'source' values - if there are multiple properties that project that value to make querying easier I don't think it should be required to list them here.

For example:
* RID is emitted - this should be documented here
* if we logged a derived value _RIDIsWindows - that shouldn't necessarily be documented.

cc @dsplaisted as privacy champ

Reply:
Sorry for my ignorance, but how I can detect which properties are source values?
Are those the properties without condition?

Reply:
I loosened up the description and in majority of cases moved away from specifying which exact MSBuild property is tracked. Especially because we mostly take values from properties which are internally set/resolved and are in most cases not really public.

This is my suggestion for the data points we collect: 
```suggestion
| >=8.0.100     | Hashed HybridGlobalization used for build (MSBuild property) |
| >=8.0.100     | Whether .NET Blazor WebAssembly SDK is used. |
| >=8.0.100     | Whether .NET WebAssembly SDK is used. |
| >=8.0.100     | Whether .NET MAUI is used. |
| >=8.0.100     | Whether .NET mobile SDK is used. |
| >=8.0.100     | Whether other mobile SDKs are used (like: Avalonia, Uno). |
| >=8.0.100     | Whether Mono AOT is used. |
| >=8.0.100     | Whether Mono AOT strip IL feature is used. |
| >=8.0.100     | Whether Mono interpreter is used. |
| >=8.0.100     | Whether library mode for mobile is used.  |
| >=8.0.100     | Whether NativeAOT is used. |
| >=8.0.100     | Used Mono runtime pack version. |
```

Reply:
Thanks @ivanpovazan , I applied your suggestions